content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Configuring Gateway Network Topology (Development)
+description: How to configure gateway network topology (Development).
+weight: 60
+keywords: [traffic-management,ingress,gateway]
+---
+
+## Configuring network topologies (Development)
+
+{{< idea >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{</ idea >}}
+
+Istio provides the ability to manage settings like [X-Forwarded-For](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for) (XFF)
+and [X-Forwarded-Client-Cert](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-client-cert)
+(XFCC), which are dependent on how the gateway workloads are deployed. This is currently an in-development feature. For more
+information on X-Forwarded-For see the IETF's [RFC](https://tools.ietf.org/html/rfc7239).
+
+You may choose to deploy Istio ingress gateways in using various network topologies
+(e.g. behind Cloud Load Balancers, a self-managed Load Balancer or directly expose the
+Istio ingress gateway to the Internet). As such, these topologies require different ingress gateway configurations for
+transporting correct client attributes like IP addresses and certificates to the workloads running in the cluster.
+
+Configuration of XFF and XFCC headers is managed via `MeshConfig` during Istio
+*installation* or by adding a pod annotation. Note that the `Meshconfig` configuration is used as a global setting for all gateway workloads where as pod annotations can be used to override the global setting on a per-workload basis.
+
+To simplify configuring network topology during installation create a single YAML file to pass to `istioctl`:
+
+{{< text yaml >}}
+cat <<'EOF' > topology.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    defaultConfig:
+      gatewayTopology:
+EOF
+{{< /text >}}
+
+{{< idea >}}
+If Istio ingress gateway was already running prior to application of the `MeshConfig` you will need
+to restart any Istio ingress gateway pods.
+{{< /idea >}}
+
+Both of the settings discussed can also be configured using the `proxy.istio.io/config` annotation to the Pod spec
+of your Istio ingress gateway.
+
+{{< text yaml >}}
+...
+  metadata:
+    annotations:
+      "proxy.istio.io/config": '{"gatewayTopology" : { "numTrustedProxies": 2 } }'
+{{< /text >}}
+
+### Configuring X-Forwarded-For Headers
+
+Applications rely on client attributes, such as `X-Forward-For` header,
+to be forwarded by reverse proxies in a request. However, due to the variety of network
+topologies Istio can be deployed in, it is required to set the number of trusted proxies deployed in front
+of the Istio gateway proxy so that the client address can be extracted correctly.
```

Comment:
```suggestion
Applications rely on reverse proxies to forward client attributes in a request, such as `X-Forward-For` header. However, due to the variety of network
topologies Istio can be deployed in, you must set the number of trusted proxies deployed in front
of the Istio gateway proxy, so that the client address can be extracted correctly.
```